ckeditor/static/ckeditor/ckeditor/config.js

```diff
@@ -7,4 +7,5 @@ CKEDITOR.editorConfig = function( config ) {
 	// Define changes to default configuration here. For example:
 	// config.language = 'fr';
 	// config.uiColor = '#AADC6E';
+	config.extraPlugins = 'autosave';
```

Comment:
I don't think we want to add this here. We don't *always* want autosave, just on instance we specify